src/search/Search.jsx

```diff
@@ -315,6 +315,7 @@ class Search extends React.Component {
                     onChange={this.onChange}
                     popupContent={popupContent}
                     disabled={disabled}
+                    aria-label="search"
```

Comment:
Maybe it's better to set `aria-label` before line305, so that users can overwrite `aria-label`